lib/realtime/data/vehicle_position.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Realtime.Data.VehiclePosition do
+  @moduledoc """
+  The subset of the VehiclePosition data in the enhanced_json feed that we use.
+  """
+  use Ecto.Schema
+
+  @type t :: %__MODULE__{
+          route_id: Realtime.Data.route_id() | nil,
+          direction: integer(),
+          label: String.t(),
+          position: Util.Position.t() | nil,
+          heading: float() | nil,
+          station_id: String.t() | nil,
+          current_status: :INCOMING_AT | :STOPPED_AT | :IN_TRANSIT_TO,
+          timestamp: DateTime.t() | nil,
+          # GTFS-RT id, e.g. "R-54826B16".
+          vehicle_id: String.t() | nil,
```

Comment:
praise: This `| nil` makes more sense, as we may still be able to show the vehicle on the ladder even if it has no GTFS-RT/OCS id.